settings.json

```diff
@@ -0,0 +1,600 @@
+            "type": "boolean",
+            "default": false
+        },
+        "MAINTENANCE_MODE": {
+            "type": "boolean",
+            "default": false
+        },
+        "MAX_PAGE_SIZE": {
+            "type": "integer",
+            "default": 1000
+        },
+        "METRICS_ENABLED": {
+            "type": "boolean",
+            "default": false
+        },
+        "NAPALM_USERNAME": {
+            "type": "string",
+            "default": ""
+        },
+        "NAPALM_PASSWORD": {
+            "type": "string",
+            "default": ""
+        },
+        "NAPALM_ARGS": {
+            "type": "object",
+            "default": {}
+        },
+        "NAPALM_TIMEOUT": {
+            "type": "integer",
+            "default": 30
+        },
+        "NETWORK_DRIVERS": {
+            "type": "object",
+            "default": {}
+        },
+        "PAGINATE_COUNT": {
+            "type": "integer",
+            "default": 50
+        },
+        "PER_PAGE_DEFAULTS": {
+            "type": "array",
+            "default": [
+                25,
+                50,
+                100,
+                250,
+                500,
+                1000
+            ]
+        },
+        "PLUGINS": {
+            "type": "array",
+            "default": []
+        },
+        "PLUGINS_CONFIG": {
+            "type": "object",
+            "default": {}
+        },
+        "PREFER_IPV4": {
+            "type": "boolean",
+            "default": false
+        },
+        "RACK_ELEVATION_DEFAULT_UNIT_HEIGHT": {
+            "type": "integer",
+            "default": 22
+        },
+        "RACK_ELEVATION_DEFAULT_UNIT_WIDTH": {
+            "type": "integer",
+            "default": 220
+        },
+        "REDIS_LOCK_TIMEOUT": {
+            "type": "integer",
+            "default": 600
+        },
+        "RELEASE_CHECK_TIMEOUT": {
+            "type": "integer",
+            "default": 86400
+        },
+        "RELEASE_CHECK_URL": {
+            "type": "string",
+            "default": ""
+        },
+        "SANITIZER_PATTERNS": {},
+        "STORAGE_BACKEND": {
+            "type": "string",
+            "default": ""
+        },
+        "STORAGE_CONFIG": {
+            "type": "object",
+            "default": {}
+        },
+        "STRICT_FILTERING": {
+            "type": "boolean",
+            "default": true
+        },
+        "SUPPORT_MESSAHE": {
+            "type": "string",
+            "default": ""
+        },
+        "TEST_FACTORY_SEED": {
+            "type": "string",
+            "default": ""
+        },
+        "TEST_USE_FACTORIES": {
+            "type": "boolean",
+            "default": false
+        },
+        "TEST_PERFORMANCE_BASELINE_FILE": {
+            "type": "string",
+            "default": "nautobot/core/tests/performance_baselines.yml"
+        },
+        "UI_RACK_VIEW_TRUNCATE_FUNCTION": {},
+        "GIT_SSL_NO_VERIFY": {},
```

Comment:
How do I express these two settings in a JSON format?

Reply:
GIT_SSL_NO_VERIFY is an environment variable, not a settings parameter, so it should be fine to omit. Not sure on the function.